morpheus/cli/commands.py

```diff
@@ -660,3 +660,5 @@ def post_pipeline(ctx: click.Context, *args, **kwargs):
 add_command("train-ae", "morpheus.stages.preprocess.train_ae_stage.TrainAEStage", modes=AE_ONLY)
 add_command("trigger", "morpheus.stages.general.trigger_stage.TriggerStage", modes=ALL)
 add_command("validate", "morpheus.stages.postprocess.validation_stage.ValidationStage", modes=ALL)
+
+add_command("from-doca", "morpheus.stages.doca.doca_source_stage.DocaSourceStage", modes=NLP_ONLY)
```

Comment:
Wont this cause an exception if someone tries to use this without doca being built?

Reply:
Yes, and that's probably ok for the time being.